moveit_planners/chomp/chomp_motion_planner/src/chomp_planner.cpp

```diff
@@ -53,33 +53,40 @@ bool ChompPlanner::solve(const planning_scene::PlanningSceneConstPtr& planning_s
 {
   if (!planning_scene)
   {
-    ROS_ERROR_STREAM("No planning scene initialized.");
+    ROS_ERROR_STREAM_NAMED("chomp_planner", "No planning scene initialized.");
     res.error_code.val = moveit_msgs::MoveItErrorCodes::FAILURE;
     return false;
   }
 
   if (req.start_state.joint_state.position.empty())
   {
-    ROS_ERROR_STREAM("Start state is empty");
+    ROS_ERROR_STREAM_NAMED("chomp_planner", "Start state is empty");
     res.error_code.val = moveit_msgs::MoveItErrorCodes::INVALID_ROBOT_STATE;
     return false;
   }
 
   if (not planning_scene->getRobotModel()->satisfiesPositionBounds(req.start_state.joint_state.position.data()))
   {
-    ROS_ERROR_STREAM("Start state violates joint limits");
+    ROS_ERROR_STREAM_NAMED("chomp_planner", "Start state violates joint limits");
     res.error_code.val = moveit_msgs::MoveItErrorCodes::INVALID_ROBOT_STATE;
     return false;
   }
 
   ros::WallTime start_time = ros::WallTime::now();
   ChompTrajectory trajectory(planning_scene->getRobotModel(), 3.0, .03, req.group_name);
+
+  // std::cout << trajectory.getTrajectory() << " 1. complete initialized TRAJECTORY in CHOMP_PLANNER..!!!!!" <<
+  // std::endl;
```

Comment:
you cleanup the dead code but then add your own commented out code? :-)

use ROS_DEBUG_STREAM_NAMED if you want to keep stuff like this. otherwise remove it

Reply:
Oh yeah, I should remove my commented code too, will remove it !